include/dfrn.php

```diff
@@ -1853,15 +1859,17 @@ private static function process_relocation($xpath, $relocation, $importer) {
 	 * @param array $item the new item record
 	 * @param array $importer Record of the importer user mixed with contact of the content
 	 * @param int $entrytype Is it a toplevel entry, a comment or a relayed comment?
+	 * @todo set proper type-hints (array?)
```

Comment:
according to the doc-tag, it is `array` which can be type-hinted. sure, I will check that when I do it, that `array` is really given/wanted.